core/src/main/scala/cats/instances/invariant.scala

```diff
@@ -1,27 +1,34 @@
 package cats.instances
 
 import cats.kernel._
-import cats.InvariantMonoidal
+import cats.{InvariantMonoidal, Monoid, InvariantSemigroupal}
 
 trait InvariantMonoidalInstances {
 
+  implicit def catsSemigroupalForMonoid: InvariantSemigroupal[Monoid] = new InvariantSemigroupal[Monoid] {
```

Comment:
I am curious why not move these two to the companion of `Monoid` and `Semigroup` to avoid a required import?

Reply:
Because they're in kernel :/

Reply:
oh Duh